.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -37,9 +61,22 @@ body:
 - type: input
   attributes:
     label: Component Name
-    description: >-
-      Write the short name of the module, plugin, task or feature below,
-      *use your best guess if unsure*.
+    description: >
+      Write the short name of the rst file, module, plugin, task or
+      feature below, *use your best guess if unsure*.
+
+
+      **Tip:** Cannot find it in this repository? Please be advised that
+      the source for some parts of the documentation are hosted outside
+      of this repository. If the page you are reporting describes
+      modules/plugins/etc that are not officially supported by the
+      Ansible Core Engineering team, there is a good chance that it is
+      coming from one of the [Ansible Collections maintained by the
+      community][collections org]. If this is the case, please make sure
+      to file an issue under the appropriate project there instead.
+
+
+      [collections org]: ../../ansible-collections
```

I think we'd want this to be absolute

```suggestion
      [collections org]: https://github.com/ansible-collections/
```

---

Why? What is the motivation behind making it verbose?

---

Just thought it makes it clearer. I didn't think this would ever need to be relative

---

@gundalow WDYT about just `/ansible-collections`? It'd be cleaner, no?